rules/validator.go

```diff
@@ -125,6 +125,11 @@ func (v *validator) validateRollupRules(rrv map[string]*RollupRuleView) error {
 		}
 
 		for _, target := range view.Targets {
+			// Validate that rollup target name is valid
```

Comment:
Might also want to extract this logic into a `valiateRollupMetricName` method, in which we can also check if the metric name is empty etc.